windows/9.3.x/sitecore-identity/Dockerfile

```diff
@@ -0,0 +1,38 @@
+#escape=`
+ARG ASSETS_IMAGE
+ARG CERTIFICATES_IMAGE
+ARG BUILD_IMAGE
+ARG BASE_IMAGE
+
+FROM ${ASSETS_IMAGE} as assets
+FROM ${CERTIFICATES_IMAGE} as certificates
+FROM ${BUILD_IMAGE} as build
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ARG ASSETS_USE_WDP
+
+COPY --from=assets ["${ASSETS_USE_WDP}", "c:\\temp\\packages\\"]
+
+RUN Expand-Archive -Path 'c:\\temp\\packages\\*.zip' -DestinationPath 'c:\\temp'
```

Comment:
I think you need to add config here XDT & license, see https://github.com/Sitecore/docker-images/blob/master/windows/9.3.x/sitecore-xp-identity/Dockerfile

Reply:
Actually, for Identity Server this can be configured entirely through environment variables thanks to [Sitecore Host](https://doc.sitecore.com/developers/93/sitecore-experience-manager/en/configuration.html#UUID-5c6cf510-2546-8c91-4393-bfa6b49a6f62_section-idm46212281838784).

So in order to set everything otherwise set by [Sitecore.IdentityServer.Host.xml.xdt](https://github.com/Sitecore/docker-images/blob/master/windows/9.3.x/sitecore-xp-identity/config/production/Sitecore.IdentityServer.Host.xml.xdt) with environment variables, you'd do

```
SITECORE_Sitecore__IdentityServer__CertificateThumbprint=<value>
SITECORE_Sitecore__IdentityServer__CertificateStoreLocation=LocalMachine
SITECORE_Sitecore__IdentityServer__CertificateStoreName=My
SITECORE_Sitecore__IdentityServer__SitecoreMembershipOptions__ConnectionString=<value>
SITECORE_Sitecore__IdentityServer__AccountOptions__PasswordRecoveryUrl=<value>
SITECORE_Sitecore__IdentityServer__Clients__DefaultClient__AllowedCorsOrigins=<value>
SITECORE_Sitecore__IdentityServer__Clients__PasswordClient__ClientSecrets__ClientSecrets=<value>
```

However, since Nano Server doesn't have the certificate store provided by Windows Server (Core), the three top lines will be pointing to a file instead:

So **instead** of these 

```
SITECORE_Sitecore__IdentityServer__CertificateThumbprint=<value>
SITECORE_Sitecore__IdentityServer__CertificateStoreLocation=LocalMachine
SITECORE_Sitecore__IdentityServer__CertificateStoreName=My
```

we'll do this intead [sitecore-identity/Dockerfile, line 35](https://github.com/anarsen/docker-images/blob/identity-server-4.0.0-nanoserver/windows/9.3.x/sitecore-identity/Dockerfile#L35).

How does this sit with you?

Reply:
Cool, didn't know that!

The environment variables settings should be added to the example docker-compose files than.

Reply:
Of course, I'll get right on it.